viewers/pydicom_Tkinter.py

```diff
@@ -147,16 +147,15 @@ def get_tkinter_photoimage_from_pydicom_image(data):
 
     # overwrite with specific values from data, if available
     if ('WindowCenter' in data) and ('WindowWidth' in data):
-        wc = data.WindowCenter
-        ww = data.WindowWidth
         try:
-            wc = wc[0]  # can be multiple values
+            ew = data['WindowWidth']
+            ec = data['WindowCenter']
+            wc = int(ew.value[0] if ew.VM > 1 else ew.value)
+            ww = int(ec.value[0] if ec.VM > 1 else ec.value)
         except Exception:
-            pass
-        try:
-            ww = ww[0]
-        except Exception:
-            pass
+            wc = (arr.max() + arr.min()) / 2.0
```

Comment:
A bare exception is not the best with a number of lines of code like this --  I assume it is meant to capture the case where WindowWidth and WindowCenter are not in `data`?  That would be better handled with the `in` operator.

Reply:
Actually, this check is already there - I think the `try/except` is just left from the previous handling of multi-valued window/center and can be removed. As far as I can see, the `except` clause shall be replaced by an `else` branch to handle the case without window/center defined.

Reply:
There is actually one case which would not be handled: if window or center has no value, e.g. VM is 0 (which should not happen, but in real life it can happen). This should be handled in the same way as if no value were given.